composer.json

```diff
@@ -1,13 +1,13 @@
 {
-    "name": "stroker/cache",
-    "description": "Provides a full page cache solution for ZF2",
+    "name": "ewilde-imperial/strokercache",
```

Comment:
Package name should still stay `stroker/cache`. I'll issue a new major version bump of this package when it is merged in master.

Reply:
Sorry, I will fix those...wasn't expecting such a quick response :)

Reply:
👍